### DISCUSSION TITLE
Should we release 0.70.7

### DISCUSSION THREAD

**Discussion by cipolleschi** [reactwg/react-native-releases#48] [Patches]
Released. Please refer to https://github.com/reactwg/react-native-releases/discussions/56

---

# Should we release 0.70.7?

Current Release: [0.70.7](https://github.com/facebook/react-native/releases/tag/v0.70.6)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1.

#### Local commits to backport to main

1.

**Comment by lukebars** [comment]
There's a misleading typo in the heading. Version should not say 0.67.1

> 
> # Should we release 0.67.1?
> 
> 
> 
> Current Release: [0.70.6]

**Comment by YangJonghun** [comment]
https://github.com/facebook/react-native/releases/tag/v0.70.6
Above change logs said `Bump version of Promise from 8.0.3 to 8.3.0, enabling Promise.allSettled and Promise.any`
but polyfills are didn't affected when hermes enabled, because 0.70.6's [hermes version](https://github.com/facebook/react-native/blob/v0.70.6/sdks/.hermesversion) use promise 8.0.3.
So, I think RN should bump promise within hermes for 0.70.0 and apply it (hermes for 0.71.0 was already bumped)
(I guess 0.69.x may have same problem, but I didn't tested)

If you need furthermore information, please check below links
- codes
  - https://github.com/facebook/hermes/blob/hermes-2022-09-14-RNv0.70.1-2a6b111ab289b55d7b78b5fdf105f466ba270fd7/utils/promise/package.json
  - https://github.com/facebook/react-native/blob/v0.70.6/Libraries/Core/polyfillPromise.js#L25
- issues
  - https://github.com/facebook/react-native/issues/35363

**Comment by dcangulo** [comment]
I think we should pick https://github.com/facebook/react-native/pull/35533/commits/28147f8812c9e2779b82d67483041048f6458f6a for 0.70.7, as discussed in https://github.com/facebook/react-native/pull/35533#pullrequestreview-1202175446

**Comment by wood1986** [comment]
Please include this tiny PR https://github.com/facebook/react-native/pull/35705

**Comment by kelset** [comment]
https://github.com/facebook/react-native/commit/be69c8b5a77ae60cced1b2af64e48b90d9955be5

**Reply by beetlebum** [reply]
Thanks Lorenzo, this bug is killing our ratings. What are the chances of getting a new RN release this week?

**Reply by kelset** [reply]
we want to first get out a nightly so that folks can test that fix in isolation, then we'll likely proceed with releases for 71-70-69 (68 TBD) early next week. We don't want to rush it out on a Friday after what happened with the maven incident.

**Comment by cjhines** [comment]
Any chance of including this small dependency change?

https://github.com/facebook/react-native/pull/35912

**Reply by kelset** [reply]
the PR is still open so we can't cherry-pick it
